atlite/__init__.py

```diff
@@ -1,29 +1,13 @@
-## Copyright 2016-2017 Gorm Andresen (Aarhus University), Jonas Hoersch (FIAS), Tom Brown (FIAS)
+# -*- coding: utf-8 -*-
 
-## This program is free software; you can redistribute it and/or
-## modify it under the terms of the GNU General Public License as
-## published by the Free Software Foundation; either version 3 of the
-## License, or (at your option) any later version.
-
-## This program is distributed in the hope that it will be useful,
-## but WITHOUT ANY WARRANTY; without even the implied warranty of
-## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-## GNU General Public License for more details.
-
-## You should have received a copy of the GNU General Public License
-## along with this program.  If not, see <http://www.gnu.org/licenses/>.
-
-
-"""
-Renewable Energy Atlas Lite (Atlite)
-
-Light-weight version of Aarhus RE Atlas for converting weather data to power systems data
-"""
```

Comment:
Did you remove the package doc string intentionally? It is what is displayed, when you import the library. And then ask for help for it, ie
```python
import atlite
?atlite
```

Instead I would like to extend it with a link to the documentation and perhaps one or two very simple examples.

Reply:
Yes, multiple reasons:

* Previous one was so out-of-date it was more confusing than helping.
* The docstring was repeated in every section of the API ref. generated by automodule on readthedocs.org - not something beautiful to look at.
* I didn't have a nice substitute.

I'd drop the examples (Pareto principle + we are working on nice examples in parallel). So I argue no need to open up a new frontier there IMHO.
Let's stick to a simple description, suggestion in the updated PR.